src/blog/2021/006-2021-year-in-review.md

```diff
@@ -0,0 +1,87 @@
+* Released in December, [`diagram-js@8`](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#800) prepares us for a [truely multi-layered future](#looking-ahead).
+
+In the larger eco-system we did significant investments in two areas:
+
+* We released new versions of [bpmnlint](https://github.com/bpmn-io/bpmnlint), our take on pluggable BPMN 2.0 diagram validation. [New validation hooks](https://github.com/bpmn-io/bpmnlint/blob/master/CHANGELOG.md#710) allow you to build more exporessive and performant lint rules; new recommended lint rules and quality of life improvements help you to get most out of the tool, out of the box. A new version of the [bpmnlint-playground](https://github.com/bpmn-io/bpmnlint-playground) makes it even easier to build and debug your lint rules in a visual manner.
+* We reworked the [bpmn-js-token-simulation](https://github.com/bpmn-io/bpmn-js-token-simulation) and turned it from an experiment into a fully fledged, visual BPMN simulator. The tool now follows the BPMN spec closely. By doing so, it allows you to not only learn the basics of token flow, but also how more complicated BPMN constructs work. [Read](https://github.com/nikku/talks/tree/main/talks/2021-token-simulation#readme) [more](https://github.com/nikku/talks/tree/main/talks/2021-token-simulation-internals#readme) on what we did, and why. [Give token simulation](https://bpmn-io.github.io/bpmn-js-token-simulation/modeler.html?e=1) a try.
+
+Significant work happened in one of the many underlying libraries that support our editing ecosystem. As an example, we [published 20 releases](https://www.npmjs.com/package/diagram-js?activeTab=versions) of [diagram-js](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md), each improving the way our core modeling UX works.
+
+We cut more than 95 releases (more than 40 being actual feature releases) in [over 30 libraries](https://github.com/bpmn-io). To communicate the new and noteworthy in our ecosystem, we published [five blog posts](https://bpmn.io/blog/) and sent out [numerous tweets](https://twitter.com/bpmn_io).
+
+
+## The Year in Numbers
+
+Similar to [previous](./2018-year-in-review.html) [years](./2019-year-in-review.html) we see significant increase in interest in our project and toolkits.
+More people visited [our project website](https://bpmn.io/), asked for help, and contributed to [our forum](https://forum.bpmn.io/) and starred our projects on [GitHub](https://github.com/bpmn-io). Community work led to several contributions, too, such as [spanish translations for bpmn-js](https://github.com/bpmn-io/bpmn-js-i18n/pull/11) and [improved](https://github.com/bpmn-io/diagram-js/pull/484) [keyboard accessibility](https://github.com/bpmn-io/diagram-js/pull/493) of our editors.
+
+<div class="figure no-border condensed-size" style="margin: 50px 0 40px 0; text-align: center">
+  <img src="{{ assets }}/attachments/blog/2021/006-stats.png" alt="Some project statistics" style="max-width: 720px">
```

Comment:
Consider to make the increased bundle size grey in colour

Reply:
Why?